CONTRIBUTING.md

```diff
@@ -68,6 +68,19 @@ If successful, it will modify `activities.json` with the new specification. Chec
 }
 ```
 
+### Asking Mozilla to Update a Position
+
+Specifications might be updated after a position is decided. Especially if those
+updates are in response to feedback provided by Mozilla. If you think that
+changes to a proposal warrant an updated position from Mozilla, please request
+another review by commenting on the original issue for the specification. Please
+include a summary of the changes that you believe to be relevant.
+
+When asking for an update you may reopen the issue. However, we ask that people
```

Comment:
I’m unsure individuals from outside can reopen issues. Maybe say: “you can ask for an issue to be reopened”

Reply:
They cannot, we should indeed rephrase that.